Makefile

```diff
@@ -375,6 +375,8 @@ helm-chart-generate: kustomize helm kubectl-slice yq charts
 	for file in $${cluster_scoped_files}; do\
 		$(YQ) -i '.metadata.name += "-{{ .Release.Name }}"' $${file};\
 	done
+	# Add tolerations and affinity to the deployment
+	$(SED_I) -E -e 's/^(      containers:)/{{- with .Values.tolerations }}\n      tolerations:\n{{- toYaml . | nindent 8 }}\n{{- end }}\n{{- with .Values.affinity }}\n      affinity:\n{{- toYaml . | nindent 8 }}\n{{- end }}\n\1/' charts/$(CHART_NAME)/raw-files/deployment-awx-operator-controller-manager.yaml
```

Comment:
I don't see this file in my checkout of the awx-operator..

Reply:
This file is created by [this command](https://github.com/ansible/awx-operator/blob/ec8c20fa8821bbb2640361b21f845e52685c9177/Makefile#L355-L358) in the Makefile. The source of that file is [config/manager/manager.yml](https://github.com/ansible/awx-operator/blob/ec8c20fa8821bbb2640361b21f845e52685c9177/config/manager/manager.yaml) but it split apart by `kustomize`.